app/modules/encounters/resources.py

```diff
@@ -123,7 +123,7 @@ def post(self, args):
                 # TODO other houston-based relationships: orgs, projects, etc
                 owner_guid = None
                 pub = True  # legit? public if no owner?
-                if current_user is not None:
+                if current_user is not None and not current_user.is_anonymous:
                     owner_guid = current_user.guid
```

Comment:
Is_anonymous always returns False. Do we need this? Do we actually need is_anonymous? Haven't really added it anywhere except coupled with a None check on current_user.

Reply:
I'd argue we completely remove the concept of `is_anonymous` from a User.

Reply:
I'm not sure that  statement is true Colin. It's true for all users that are logged in but for users that are not, is_anonymous  definitely does return True

Reply:
https://github.com/WildMeOrg/houston/blob/develop/app/modules/users/models.py#L442

Reply:
We need to keep `is_anonymous` to support Flask-Login: https://flask-login.readthedocs.io/en/latest/#your-user-class